src/main/java/api/WhisperTranscription.java

```diff
@@ -14,11 +14,10 @@
 
 public class WhisperTranscription implements TranscriptionInterface {
 
-    final private String OPENAI_API_KEY;
+    final private String OPENAI_API_KEY="";
     final private int MAX_CHUNK_SIZE_BYTES = 20 * 1024 * 1024;
 
-    public WhisperTranscription(String apiKey) {
-        OPENAI_API_KEY = apiKey;
+    public WhisperTranscription() {
```

Comment:
I don't think we should hard-code the api key into the source, because then we can't push to github. We need to retrieve it from a file in the Main method, so I think we should revert this change.

Reply:
Yes we don't have to hard code it but I think we should retrieve it from a file and set it as default just so we don't need to pass it in everytime we call the api. Let me know what you think about this approach.

Reply:
We can get the api key in the `main` method and then input it when we create the object. Putting it in the class itself is not good, because it makes it specific to our uses, rather than being general.